litrpc/lit-sessions.proto

```diff
@@ -15,11 +15,12 @@ service Sessions {
 }
 
 enum SessionType {
+    reserved 4;
```

Comment:
nit: keep this in the original order? So between 3 and 5? Or does the compiler complain then?

Reply:
done 👍